manifests/Epsitec/Colobot/0.1.12.yaml

```
@@ -0,0 +1,14 @@
+Id: Epsitec.Colobot
+Version: 0.1.12
+Name: "Colobot: Gold Edition"
+Publisher: Epsitec SA, TerranovaTeam
```

```suggestion
Publisher: Epsitec
```
the parent folder should be as same as the Publisher

---

Is this really the issue? The pull request I submitted for Planet Blupi, another title by Epsitec SA, checked out fine: https://github.com/microsoft/winget-pkgs/pull/770

Other manifests, such as [`EpicGames.EpicGamesLauncher`](https://github.com/microsoft/winget-pkgs/blob/master/manifests/EpicGames/EpicGamesLauncher/10.15.2.Yaml), [`GitHub.cli`](https://github.com/microsoft/winget-pkgs/blob/master/manifests/GitHub/cli/0.8.0.yaml) and Microsoft's own [`Microsoft.Edge`](https://github.com/microsoft/winget-pkgs/blob/master/manifests/Microsoft/Edge/81.0.416.77.yaml) also have a publisher that's different from the parent folder.

Looking at the [pipeline](https://dev.azure.com/ms/winget-pkgs/_build/results?buildId=81962&view=logs&jobId=c8020aa5-4aab-5881-691f-21986e02afd0&j=c8020aa5-4aab-5881-691f-21986e02afd0&t=bade86d6-d2be-5ddd-c1b9-7d8db9c4a60d) it appears that the "Installers Scan" task simply ends up timing out.

---

oh you are right. sorry for my wrong suggestion